util/src/main/java/org/datacommons/util/ApiHelper.java

```
@@ -45,16 +45,19 @@ public static JsonObject fetchPropertyValues(
     }
 
     JsonObject arg = new JsonObject();
-    arg.add("dcids", dcids);
-    arg.addProperty("property", property);
-    arg.addProperty("direction", "out");
+    arg.add("nodes", dcids);
+    // V2 uses -> for out-edges, which is equivalent to direction: "out" in V1
+    arg.addProperty("property", "->" + property);
 
-    var request =
+    var requestBuilder =
         HttpRequest.newBuilder(URI.create(API_ROOT))
             .version(HttpClient.Version.HTTP_1_1)
             .header("accept", "application/json")
-            .POST(HttpRequest.BodyPublishers.ofString(arg.toString()))
-            .build();
+            .POST(HttpRequest.BodyPublishers.ofString(arg.toString()));
+    if (API_KEY != null && !API_KEY.isEmpty()) {
```

Do we need to throw an exception if API_KEY is not set?